### PR TITLE
update weather handler to use new sessions

### PR DESCRIPTION
**Pull request by guynir42**
Use the new baselayer session to update the weather handler.

### CODE REVIEW

**Comment by guynir42** [comment]
This should only be reviewed/merged after the baselayer SQLA2.0 PR goes in.

**Comment by stefanv** [comment]
Depends on https://github.com/cesium-ml/baselayer/pull/312

**Comment by mcoughlin** [comment]
@guynir42 Do you want to merge in main and we can get this in?

**Comment by guynir42** [comment]
@mcoughlin only 220 commits since I last worked on this. 
And no conflicts!

**Comment by mcoughlin** [comment]
Doesn't anybody care about the weather?

File: skyportal/handlers/api/weather.py [hunk 1]
```diff
@@ -66,72 +67,75 @@ def get(self):
...
+                response = get_url(
+                    "https://api.openweathermap.org/data/2.5/onecall?"
+                    f"lat={telescope.lat}&lon={telescope.lon}&appid={openweather_api_key}"
+                )
+                if response is not None:
+                    if response.status_code == 200:
+                        data = response.json()
+                        weather.weather_info = data
+                        weather.retrieved_at = datetime.datetime.utcnow()
+                        session.verify()
```
Comment (mcoughlin) [inline skyportal/handlers/api/weather.py:124]: Why is verify needed here?

File: skyportal/handlers/api/weather.py [hunk 2]
```diff
@@ -66,72 +67,75 @@ def get(self):
...
+                        session.commit()
```
Comment (mcoughlin) [inline skyportal/handlers/api/weather.py:125]: Can it just be the commit below?

**Review by mcoughlin — APPROVED** [review]
@guynir42 LGTM!

**Review by mcoughlin — APPROVED** [review]
LGTM

**Pull request merged**